2015-04-10 01:33:04.925 [error] <0.97.0> gen_server boss_db_mock terminated with reason: no function clause matching boss_db_mock_controller:match_cond({greeting,"greeting-1","Hello, World!"}, [{greeting_text,gtt,"^Hellespont"}]) line 121
2015-04-10 01:33:04.926 [error] <0.97.0> CRASH REPORT Process <0.97.0> with 0 neighbours exited with reason: no function clause matching boss_db_mock_controller:match_cond({greeting,"greeting-1","Hello, World!"}, [{greeting_text,gtt,"^Hellespont"}]) line 121 in gen_server:terminate/7 line 804
2015-04-10 01:33:04.926 [error] <0.96.0> Supervisor {global,boss_db_mock_sup} had child db_mock_controller started with boss_db_mock_controller:start_link([]) at <0.97.0> exit with reason no function clause matching boss_db_mock_controller:match_cond({greeting,"greeting-1","Hello, World!"}, [{greeting_text,gtt,"^Hellespont"}]) line 121 in context child_terminated
2015-04-10 01:33:04.927 [error] <0.103.0> gen_server <0.103.0> terminated with reason: {{function_clause,[{boss_db_mock_controller,match_cond,[{greeting,"greeting-1","Hello, World!"},[{greeting_text,gtt,"^Hellespont"}]],[{file,"src/boss_db_mock_controller.erl"},{line,121}]},{dict,filter_bucket,4,[{file,"dict.erl"},{line,475}]},{dict,filter_bkt_list,4,[{file,"dict.erl"},{line,469}]},{dict,filter_seg_list,4,[{file,"dict.erl"},{line,463}]},{dict,filter_dict,2,[{file,"dict.erl"},{line,458}]},{boss_db_mock_controller,do_find,7,[{file,"src/boss_db_mock_controller.erl"},{line,112}]},...]},...} in gen_server:call/2 line 182
2015-04-10 01:33:04.927 [error] <0.103.0> CRASH REPORT Process <0.103.0> with 0 neighbours exited with reason: {{function_clause,[{boss_db_mock_controller,match_cond,[{greeting,"greeting-1","Hello, World!"},[{greeting_text,gtt,"^Hellespont"}]],[{file,"src/boss_db_mock_controller.erl"},{line,121}]},{dict,filter_bucket,4,[{file,"dict.erl"},{line,475}]},{dict,filter_bkt_list,4,[{file,"dict.erl"},{line,469}]},{dict,filter_seg_list,4,[{file,"dict.erl"},{line,463}]},{dict,filter_dict,2,[{file,"dict.erl"},{line,458}]},{boss_db_mock_controller,do_find,7,[{file,"src/boss_db_mock_controller.erl"},{line,112}]},...]},...} in gen_server:terminate/7 line 804
2015-04-10 01:33:04.928 [error] <0.100.0> Supervisor {<0.100.0>,poolboy_sup} had child boss_db_controller started with {boss_db_controller,start_link,undefined} at <0.103.0> exit with reason {{function_clause,[{boss_db_mock_controller,match_cond,[{greeting,"greeting-1","Hello, World!"},[{greeting_text,gtt,"^Hellespont"}]],[{file,"src/boss_db_mock_controller.erl"},{line,121}]},{dict,filter_bucket,4,[{file,"dict.erl"},{line,475}]},{dict,filter_bkt_list,4,[{file,"dict.erl"},{line,469}]},{dict,filter_seg_list,4,[{file,"dict.erl"},{line,463}]},{dict,filter_dict,2,[{file,"dict.erl"},{line,458}]},{boss_db_mock_controller,do_find,7,[{file,"src/boss_db_mock_controller.erl"},{line,112}]},...]},...} in context child_terminated
